### PR TITLE
8.5 release notes

### PR DESCRIPTION
**Pull request by akeller**
## Description

Introducing new release notes format via https://github.com/camunda/product-hub/issues/1901

image

## When should this change go live?

- [ ] This change is not yet live and should not be merged until {ADD_DATE} (apply `hold` label or convert to draft PR)?
- [ ] There is no urgency with this change.
- [ ] This change or page is part of a marketing blog, conference talk, or something else on a schedule.
- [x] This functionality is already available but undocumented.
- [ ] This is a bug fix or security concern.

## PR Checklist

- [x] I have added changes to the relevant `/versioned_docs` directory, or they are not for an **already released version**.
- [x] I have added changes to the main `/docs` directory (aka `/next/`), or they are not for **future versions**.
- [x] My changes require an [Engineering review](https://github.com/camunda/camunda-docs/blob/main/howtos/documentation-guidelines.md#review-process), and I've assigned an engineering manager or tech lead as a reviewer, or my changes do not require an Engineering review.
- [x] My changes require a [technical writer review](https://github.com/camunda/camunda-docs/blob/main/howtos/documentation-guidelines.md#review-process), and I've assigned @christinaausley as a reviewer, or my changes do not require a technical writer review.

### CODE REVIEW

**Comment by akeller** [comment]
@MoAtCamunda this PR includes:

* Release notes is now a folder with a category doc link (when you click release notes, the main page opens)
* 850.md holds 8.5.0 and all alphas
  * page includes similar format to Zeebe release notes - enhancements, bug fixes, maintenance (do you like? yay or nay? or just stick to the single section for now?)

Tomorrow (if time allows) I will add two real examples from the 8.5 scoped list.

**Comment by akeller** [comment]
Changes after C&P from Product Hub, without edits - 
image

**Comment by akeller** [comment]
@christinaausley @MoAtCamunda I did a C&P from the Product Hub and some light edits for casing, product names, and acronyms. Please review what's in the PR.

I'm hoping to add another commit to includes some links either to the docs on the specific epic or the public epic itself. Note, not all epics mentioned have a corresponding public epic. We can easily follow that up later this week.

**Comment by akeller** [comment]
@MoAtCamunda please review

File: docs/reference/release-notes/850.md [hunk 1]
```diff
@@ -0,0 +1,188 @@
+---
+id: 850
+title: "8.5 Release notes"
+description: "Release notes for 8.5, including alphas"
+---
+
+## Enhancements
+
+### Role Support for Web Modeler with OIDC in SM
```
Comment (christinaausley) [inline docs/reference/release-notes/850.md:9]: Maybe this is another backlog item for me as I'm not sure if we do it anywhere else, but I'd probably avoid abbreviations of SM 👍

File: docs/reference/release-notes/850.md [hunk 2]
```diff
@@ -0,0 +1,188 @@
...
+
+You can now use Role-Based Access Control with your own OIDC Identity Provider (i.e. Entra ID) and the Web Modeler without relying on Keycloak. This extends RBAC and Role Mapping support that is available for other components to Web Modeler.
+
+### Introductory UI header for processes page
+
+The process overview page was redesigned for clarity and ease-of-use.
+
+### Fully supported Spring Zeebe SDK
+
+Use supported Spring Zeebe SDK in your Spring or Spring Boot projects to interact with Camunda 8 engine API and build process applications. Spring Zeebe SDK works with Zeebe gRPC and the new Zeebe REST API.
```
Comment (christinaausley) [inline docs/reference/release-notes/850.md:19]: What do we mean by "Camunda 8 engine API"?

File: docs/reference/release-notes/850.md [hunk 3]
```diff
@@ -0,0 +1,188 @@
...
+
+### KPI Report Wizard Framework
+
+The KPI Wizard is the new way to build a KPI Report quickly. The Wizard already contains predefined KPIs that can be selected. It guides the users through the needed customization and allows them to activate an alert that allows passive monitoring. The list of predefined KPIs will be extended with upcoming releases. Start setting up a first Process KPI now.
+
+### Use Secrets in Hybrid Mode
+
+You are now able to consume the secrets from the console in your own custom connector which is connected to our SAAS platform.
+
+### Multi-Region: Productise the 2 Regions, Active-Active, Self-Managed setup (Zeebe, Operate, Tasklist)
```
Comment (christinaausley) [inline docs/reference/release-notes/850.md:95]: I think this header is confusing me a bit. Maybe "Run Camunda in two regions synchronously with a multi-region setup"?

File: docs/reference/release-notes/850.md [hunk 4]
```diff
@@ -0,0 +1,188 @@
...
+
+You can now run Zeebe, Operate and Tasklist across 2 geographical regions. You will find installation and failover guides to help you recover quickly and safely from whole region failures.
+
+Note that there is no dedicated guides how to setup Optimize, Identity and Connectors across regions yet.
+
+### REST Postman Collection to Connector Template Converter
+
+Via a command line tool you can now generate your connector template based on our REST Connector from a postman collection. We are supporting multiple operations + splitting up the body to single fields
+
+### SOAP SDK Support (SM + Hybrid)
```
Comment (christinaausley) [inline docs/reference/release-notes/850.md:105]: Same comment here on SM.

File: docs/reference/release-notes/850.md [hunk 5]
```diff
@@ -0,0 +1,188 @@
...
+
+You can now connect to a SOAP service via our Camunda Connector. The first iteration of this connector will be only available in SM and as hybrid setup
+
+### Process Blueprint Import via Link
+
+You can directly use process blueprints from our marketplace in your SAAS Web Modeler. Just click on "For SAAS". The blueprint will be downloaded to your web modeler and is ready to use.
```
Comment (christinaausley) [inline docs/reference/release-notes/850.md:111]: Is the UI really written out as "For SAAS" or "For SaaS"?

**Review by christinaausley — COMMENTED** [review]
A couple non-blocking comments for clarity.

The only other thing I would like to see is links to all of the relevant documentation for each topic. If users see the release notes first, I'm wondering if they'll feel like they need to dig through the docs to find the referenced content.

**Review by MoAtCamunda — DISMISSED** [review]
Approved - however I would like to see docs links for each if possible? Happy for this to be a separate PR tho

**Comment by akeller** [comment]
Let's merge this as-is, and I will follow up with links since that will be a bit more manually intensive. 

@MoAtCamunda, if we want to script this from the PDP in the future, we need to figure out how to make this information (the links, not just the docs PRs) easy to find.

File: docs/reference/release-notes/850.md (hunk 5 above)
Comment (akeller) [reply docs/reference/release-notes/850.md]: Good catch, not sure.

File: docs/reference/release-notes/850.md (hunk 1 above)
Comment (akeller) [reply docs/reference/release-notes/850.md]: Oops, I thought I caught all of them - let's definitely adjust this.

Comment (akeller) [reply docs/reference/release-notes/850.md]: Fixed

File: docs/reference/release-notes/850.md (hunk 3 above)
Comment (akeller) [reply docs/reference/release-notes/850.md]: I agree, although I think the Active-Active language is something we use in customer conversations. I'll make a note to follow up on this.

File: docs/reference/release-notes/850.md (hunk 4 above)
Comment (akeller) [reply docs/reference/release-notes/850.md]: Fixed

File: docs/reference/release-notes/850.md (hunk 5 above)
Comment (akeller) [reply docs/reference/release-notes/850.md]: image

Confirmed, **For SAAS**

File: docs/reference/release-notes/850.md (hunk 2 above)
Comment (akeller) [reply docs/reference/release-notes/850.md]: I _think_ this is a reference to the core components vs. Web Modeler, for example. I will follow up.

**Comment by akeller** [comment]
@pepopowitz 2 quick questions -

1. I don't think this needs a redirect right? Because the main **Release notes** page URL still resolves to http://localhost:3000/docs/next/reference/release-notes/

2. Can I get a 👍 from you so I don't go feral DRI and merge it myself? 😈

**Comment by pepopowitz** [comment]
> @pepopowitz 2 quick questions -

> I don't think this needs a redirect right? Because the main Release notes page URL still resolves to http://localhost:3000/docs/next/reference/release-notes/

I think this is correct, also!

**Review by pepopowitz — APPROVED** [review]
🚢

**Comment by christinaausley** [comment]
@akeller Happy to apply the links in a separate PR if you'd like!

**Comment by akeller** [comment]
@christinaausley see my post - https://camunda.slack.com/archives/C026U8GBNSW/p1713198186536799

We ultimately want to automate this, so we need to keep the info in Product Hub epics. Once we get some feedback we can decide how we want to move forward with the links.

File: docs/reference/release-notes/850.md (hunk 2 above)
Comment (akeller) [reply docs/reference/release-notes/850.md]: Follow up here - https://camunda.slack.com/archives/C02AWA0RF8A/p1713198528757339

File: docs/reference/release-notes/850.md (hunk 3 above)
Comment (akeller) [reply docs/reference/release-notes/850.md]: Follow up here - https://camunda.slack.com/archives/C02AWA0RF8A/p1713198528757339

File: docs/reference/release-notes/850.md (hunk 2 above)
Comment (aleksander-dytko) [reply docs/reference/release-notes/850.md]: By `Camunda 8 engine API` I meant Zeebe API - we can change it to Zeebe to me it easier to understand

Comment (akeller) [reply docs/reference/release-notes/850.md]: Will do!

Comment (akeller) [reply docs/reference/release-notes/850.md]: https://github.com/camunda/camunda-docs/pull/3696

**Pull request merged**